### ISSUE TITLE
`get_friends()` and `get_followers()` return objects with different column names.

### ISSUE THREAD

**Issue by simonheb** [ropensci/rtweet#594]
### Problem

The two functions `get_friends()` and `get_followers()` seem to be working very similarly but return different objects, in particular `get_friends()` returns a data.frame with the columns `user` and `ids. While `get_followers()` returns the a data.frame with the column `user_id`.

### Expected behavior
Users might expect  `get_friends()`  to return a column named `user_ids` for consistency with `get_followers()`

### Reproduce the problem

``` r
my_followers <- get_followers("CaptionClerk")
my_friends <- get_friends("CaptionClerk")
print(names(my_followers))
print(names(my_friends))
```

### rtweet version
‘0.7.0.9000’

**Comment by llrs** [comment]
I think this is addressed by #572 but I haven't checked. When that PR is merged all functions will return the same structure (Except when they return users data).

**Comment by llrs** [comment]
Sorry, I forgot to mention, this was also discussed on #308 and solved at https://github.com/ropensci/rtweet/commit/1c6025b84c2fcbb72e41ae969cc7060ce0c99bc8. Together with https://github.com/ropensci/rtweet/commit/5dd07d8f6c04e49a8a43e8ff87634963adef370f the returned objects should always be consistent.
